README.md

```diff
@@ -29,16 +29,29 @@ You can deploy the bot in a simple way to Heroku using the button below. [Click
 
 <!-- [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy) -->
 
-[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://dashboard.heroku.com/new?button-url=https://github.com/synzen/Discord.RSS-Clone&template=https://github.com/synzen/Discord.RSS-Clone/tree/dev)
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://dashboard.heroku.com/new?button-url=https://github.com/synzen/Discord.RSS-Clone&template=https://github.com/synzen/Discord.RSS-Clone/tree/master)
 
 *If you want to deploy manually you can [follow this guide](https://github.com/synzen/Discord.RSS/issues/95).*
 
 ### Hosting on Glitch
 
-Click the button below to import the code to Glitch. Wait for the importing process to complete, make your project private and fill in the [Configuration](https://github.com/synzen/Discord.RSS/wiki/Configuration). Use a tool like [Uptime Robot](https://uptimerobot.com/) to prevent the bot from going offline.
+Click the button below to import the code to Glitch. Wait for the importing process to complete, make your project private and fill in the [Configuration](https://docs.discordrss.xyz/configuration/bot-configuration). Use a tool like [Uptime Robot](https://uptimerobot.com/) to prevent the bot from going offline.
 
 [![Remix on Glitch](https://cdn.glitch.com/2703baf2-b643-4da7-ab91-7ee2a2d00b5b%2Fremix-button.svg)](https://glitch.com/edit/#!/import/github/synzen/Discord.RSS-Clone)
 
 You can use [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) for the MongoDB database and [Redis Labs](https://redislabs.com/) for the Redis database.
 
 The web port for Glitch is 3000.
+
+You will also need to modify the scripts section of the package.json in order for the bot to work properly on Glitch.
+
+Replace the lines starting from `"scripts": {` and ending in `},` with:
+
+```
+  "scripts": {
+    "preinstall": "npm install --save node@12.16.3",
+    "migrate-v6": "node migrations/v6.js",
+    "start": "node bot-web.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+```
```

Comment:
I'm a bit wary of replacing the *entire* scripts object since that may be subject to change, it's probably better if we just notify users they just have to add the line 
```
"preinstall": "npm install --save node@12.16.3",
```
at the beginning

Reply:
Putting server.js does not start the web UI as far as I know, it has to be changed to bot-web.js for both bot and web to work correctly on Glitch. So adding that instruction as well.